setup.py

```diff
@@ -283,17 +298,14 @@ def fully_define_extension(build_ext):
         nccl_include_dirs = nccl_lib_dirs = []
 
     MACROS = []
-    INCLUDES = tf_include_dirs
+    INCLUDES = []
     SOURCES = ['horovod/tensorflow/mpi_message.cc',
                'horovod/tensorflow/mpi_ops.cc',
                'horovod/tensorflow/timeline.cc']
-    COMPILE_FLAGS = ['-std=c++11', '-fPIC', '-O2'] + shlex.split(mpi_flags)
-    LINK_FLAGS = shlex.split(mpi_flags)
-    LIBRARY_DIRS = tf_lib_dirs
-    LIBRARIES = tf_libs
-
-    if tf_abi:
-        COMPILE_FLAGS += ['-D%s=%s' % tf_abi]
+    COMPILE_FLAGS = ['-std=c++11', '-fPIC', '-O2'] + shlex.split(mpi_flags) + tf_compile_flags
+    LINK_FLAGS = shlex.split(mpi_flags) + tf_link_flags
+    LIBRARY_DIRS = []
+    LIBRARIES = []
```

Comment:
LIBRARY_DIRS/LIBRARIES are empty because the they are included in the link_flags, right?

Reply:
Yes. However, if GPU support is enabled, CUDA libraries will be added there.